typedargs/doc_parser.py

```diff
@@ -321,14 +321,83 @@ def wrap_and_format(self, width=None, include_params=False, include_return=False
         return out.getvalue()
 
 
+def _parse_param_validators(param_desc):
+    """Get validators from parameter description.
+
+    Validators should be specified in braces. Arguments of validator should be in brackets.
+    Example of param_desc value:
+
+        {nonnegative, range(1, 5), list(['a', 'b'], istrue(True))} Parameter descriptiom
+
+    Validator arguments in parenthesis should be valid python code.
+    Allowed validator argument types are: str, int, bool, float, None, list
+
+    Args:
+        param_desc (str): parameter description
```

Comment:
Type and return value should be moved to function annotations.

Reply:
done 2dab869